contracts/PoolManager.sol

```diff
@@ -108,6 +108,25 @@ contract PoolManager is IPoolManager, Fees, NoDelegateCall, ERC1155, IERC1155Rec
         return pools[id].positions.get(owner, tickLower, tickUpper);
     }
 
+    function _accountNoOp(PoolKey memory key, BalanceDelta hookDelta) internal {
+        // NOTE: we are in the lock state of the user, not the hook, unless the hook is the user
+        _accountDelta(key.currency0, hookDelta.amount0());
```

Comment:
nit: why not just use `_accountPoolBalanceDelta` for the whole hookDelta?

Reply:
updated in most current commit, but it increases gas for modifyPosition by 56 and swap by 17